internal/session/sqlite_store.go

```
@@ -0,0 +1,306 @@
+
+func NewSQLiteStore(dbPath string, ttl time.Duration) (*SQLiteStore, error) {
+	db, err := sql.Open("sqlite", dbPath)
+	if err != nil {
+		return nil, fmt.Errorf("open sqlite: %w", err)
+	}
+
+	for _, pragma := range []string{
+		"PRAGMA journal_mode=WAL",
+		"PRAGMA foreign_keys=ON",
+	} {
+		if _, err := db.Exec(pragma); err != nil {
```

golangci-lint (2.12.1)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)